### PR TITLE
Importing project from other git

### PR DESCRIPTION
**Pull request by KamehaMehdi**

### CODE REVIEW

**Comment by stanislas14789** [comment]
Ok

**Review by stanislas14789 — APPROVED** [review]
Ok

File: app-dashboard/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "nextjs",
+  "version": "4.0.0",
```
Comment (Themimitoof) [inline app-dashboard/package.json]:
```suggestion
  "version": "0.1.0",
```

**Review by Themimitoof — APPROVED** [review]
OK but please change now (or one the next PR what I purposes).

**Pull request merged**